var/spack/repos/builtin/packages/r-cn-mops/package.py

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RCnMops(RPackage):
+    """cn.mops (Copy Number estimation by a Mixture Of PoissonS) is a data
+    processing pipeline for copy number variations and aberrations (CNVs and
+    CNAs) from next generation sequencing (NGS) data."""
+
+    homepage = "https://www.bioconductor.org/packages/cn.mops/"
+    url      = "https://www.bioconductor.org/packages/release/bioc/src/contrib/cn.mops_1.26.0.tar.gz"
+
+    version('1.26.0', 'e95b8477a957fcf5547ffb936cbeda39')
+
+    depends_on('r@2.12:', type=('build', 'run'))
+    depends_on('r-genomicranges', type=('build', 'run'))
```

Comment:
I _think_ this is missing a bunch of dependencies. See the "Imports" section under details:
https://bioconductor.org/packages/release/bioc/html/cn.mops.html